### DISCUSSION TITLE
Channel settings are a too long form

### DISCUSSION THREAD

**Discussion by robsonsobral** [ExpressionEngine/ExpressionEngine#2911] [Feature Requests]
The channel settings form are soooooooooooo long that get confusing to find just one setting to edit. Is it possible to split it into multiple parts of an accordion?

**Comment by intoeetive** [comment]
As of EE6, channel settings page is using tabs for different parts of it

**Comment by robsonsobral** [comment]
Oh, I was talking about version 6. URLs, comments, etc... It's so many stuff and so many similar titles.

**Comment by robsonsobral** [comment]
This is what I'm talking about, @intoeetive .

![image](https://user-images.githubusercontent.com/931642/127551006-d9920aef-b286-481d-995f-93c230f09901.png)

**Comment by robsonsobral** [comment]
@ajohnson , please, keep on mind.

**Comment by ajohnson** [comment]
@robsonsobral Just catching up with this one. Let me put some thought into this one. I might have an idea that could help.

**Comment by intoeetive** [comment]
@TomJaeger I think it's worth splitting channel settings into several tabs, should make navigation a lot easier. This will add like 5 extra tabs, but I think it's completely worth that
